packages/create-react-app-demo/package.json

```diff
@@ -12,7 +12,7 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
-    "ai-jsx": "0.2.0-4",
+    "ai-jsx": "file:../ai-jsx",
```

Comment:
Ah, is this the better way to do the monorepo than needing to make sure the versions line up?

Reply:
Yes, but it does not always work for some reason - reverting this change for now.